docs/src/helpers/tokens-helper.js

```diff
@@ -107,40 +107,49 @@ export const getTokenValue = (token: Token, platform: Platform): string => {
   return value || '-';
 };
 
-export const tokenSortFn = (token: { name: string, value: string }) => {
-  if (token.name.indexOf('_XXS') !== -1) {
+export const tokenNameToSizeNumber = (token: {
+  name: string,
+  value: string,
+}) => {
+  const nameIncludes = searchTerm =>
+    token.name.toLowerCase().includes(`_${searchTerm}`);
+
```

Comment:
Can we move this outside the function so that it's not created every time?

Reply:
Done